data/showcase/les_meves_ajudes.yml

```diff
@@ -2,7 +2,7 @@ title: Les meves ajudes
 description:
   fr: Les meves ajudes utilise OpenFisca et sa Web API pour informer les citoyens de Barcelone de leur éligibilité aux aides sociales.
   en: Les meves ajudes uses OpenFisca and its Web API to inform Barcelona citizens on their eligibility to social benefits.
-package: barcelona
+package: ES-B
```

Comment:
I had understood we would use the `name` of the package in the reuses, as it is what is most understandable for reusers…? 🤔 If that is not what you had understood, let's talk about it 😉